src/System.Net.Requests/src/System/Net/HttpDateParse.cs

```diff
@@ -398,5 +398,8 @@ public static bool ParseHttpDate(string dateString, out DateTime result)
 
             return true;
         }
+
+        public static DateTime StringToDate(string S) =>
```

Comment:
nit: The string parameter to this method, "S", should be lowercase "s".

Reply:
I misread it and thought you meant the type name...